src/components/Rating/Rating.visual.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { VisualContainerElement } from '../../../test/visual/VisualContainerElement';
+import { Rating } from './';
+
+class RatingVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
+        <VisualContainerElement>
+          <Rating {...this.props} />
+        </VisualContainerElement>
+      </TPAComponentsProvider>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+      {
+        it: 'mobile',
+        props: {
+          mobile: true,
```

Comment:
Again we can probably remove reference for `mobile` state.
In addition we should probably add tests showing different selected `values`, and states (`disabled`, `error`).